cmd/priv_val_server/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	cmn "github.com/tendermint/tmlibs/common"
+	"github.com/tendermint/tmlibs/log"
+
+	priv_val "github.com/tendermint/tendermint/types/priv_validator"
+)
+
+func main() {
+	var (
+		chainID     = flag.String("chain-id", "mychain", "chain id")
+		numClients  = flag.Int("clients", 1, "number of concurrently connected clients")
```

Comment:
how am I supposed to know the number of clients beforehand?

Reply:
Or this is a maximum number of concurrent clients?

Reply:
Yes, will make the description more clear.